api/Makefile

```diff
@@ -40,7 +40,7 @@ vendor:
 .PHONY: test
 test: tidy
 	@echo "Running tests..."
-	gotest -race -short -cover -coverprofile cover.out ${SRC_ROOT}/... -tags integration
+	gotest -gcflags=-l -race -short -cover -coverprofile cover.out ${SRC_ROOT}/... -tags integration
```

Comment:
This new flag is needed because the package `bou.ke/monkey` we're using to patch certain functions called in our e2e tests is not able to properly patch certain functions. Essentially this flag turns off [inlining](https://dave.cheney.net/2020/05/02/mid-stack-inlining-in-go), which is what's recommended by the package's writer [here](https://github.com/bouk/monkey?tab=readme-ov-file#notes).

While we're on this, I found out that this patching package is apparently super old and there are probably newer alternatives that we might consider in the future 🤔

Reply:
its archived since 2020 😓